lib/tasks/live_content_report.rake

```diff
@@ -0,0 +1,38 @@
+require 'csv'
+require 'tempfile'
+
+desc 'Generates a CSV report of all live documents for the supplied publishing app(s)'
+task :live_content_report, [] => :environment do |_, args|
```

Comment:
Given the size of this Rake task, I would be tempted to move it into a `ContentReportExporter` class in `lib` which does all this and then it can be easily tested.

Reply:
I thought it was roughly equivalent to existing rake tasks, but rubocop seems to agree with you `lib/tasks/live_content_report.rake:5:1: C: Metrics/BlockLength: Block has too many lines. [28/25]`, so I'll do a thing.

Reply:
Did a rebase where I extracted code from the rake file to an object in lib that can be tested.